test/Testing.js

```diff
@@ -0,0 +1,153 @@
+
+// var db = require('./db');
+// var heroesClass = require('./heroes');
+// var assert = require('assert');
+
+const chai = require('chai'),
+	chaiHttp = require('chai-http'),
+	mocha = require('mocha'),
+	sinon = require('sinon'),
+	db = require('../db'),
+	heroesClass = require('../heroes'),
+	server = require('../server'),
+	assert = chai.assert;
+
+chai.should();
+chai.use(chaiHttp);
+
+describe('Heroes', function()
+{
+	describe('#getHeroes()', function()
+	{
+		it('should return all heroes', function()
+		{
+			const heroes = new heroesClass.Heroes();
+			assert.equal(db.heroes,heroes.getHeroes());
+		});
+	});
+
+	describe('#getHeroById()', function()
+	{
+		it('should return hero with same id', function()
+		{
+			const heroes = new heroesClass.Heroes();
+			assert.equal(db.heroes.find(element => element.id === 5),heroes.getHeroById(5));
+		});
+	});
+
+	describe('#addHero()', function()
+	{
+		it('should add hero', function()
+		{
+			const heroes = new heroesClass.Heroes();
+			const hero = {id:15,name:"lol"};
+			heroes.addHero(hero);
+			assert.equal(db.heroes.find(element => element.id === 15),hero);
+		});
+	});
+
+	describe('#deleteHeroById()', function()
+	{
+		it('should delete hero with same id', function()
+		{
+			const heroes = new heroesClass.Heroes();
+			heroes.heroesDB = [{id:15,name:"lol"},{id:11,name:"df"}];
+			heroes.deleteHeroById(15);
+			const heroAfterDelete = heroes.heroesDB.find(element => element.id ===15);
+			assert.equal(heroAfterDelete,undefined);
+		});
+	});
+
+	describe('#updateHeroName()', function()
+	{
+		it('should update hero name ', function()
+		{
+			const heroes = new heroesClass.Heroes();
+			heroes.heroesDB = [{id:15,name:"lol"},{id:11,name:"df"}];
+			const heroBeforeUpdate = heroes.heroesDB.find(element => element.id ===15);
+			let hero = {id:15,name:"lol"};
+			heroes.updateHeroName(hero);
```

Comment:
whats the update here?
hero with id 15 initialized to name 'lol' and you update him with same data
update function can be broke and this test will still pass